README.md

```diff
@@ -19,18 +41,23 @@ To test the static file handler, you can try requesting some of the following re
 * `localhost:8080/static1/index.html`
 * `localhost:8080/static2/ducks.jpg`
 
+---
+
 ### Running tests
 * `make integration` runs the integration test only
 * `make test` builds the test binaries for the individual files. You can then run each test separately, if you choose (for example, `./request_handler_echo_test`)
 * `make all-tests` builds and runs all unit and integration tests
 
+---
 
-###Dealing with bound ports
+### Dealing with bound ports
 If when running the server you get an error that the port is already bound,
 execute `sudo lsof -i:8080` (change 8080 to whatever port number you're using).
 Then find the PID of the process associated with the server and kill it using `sudo kill -9 <pid>`
```

Comment:
I'm pretty sure that CTRL-Z'ing while running the server was causing this, CTRL-C would always kill the process. In any case, this wont be a problem now that the server is deployed